fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java

```diff
@@ -279,8 +279,7 @@ protected void init() {
         }
         dbNameToId = tmpDbNameToId;
         idToDb = tmpIdToDb;
-        long currentTime = System.currentTimeMillis();
-        lastUpdateTime = currentTime;
+        lastUpdateTime = System.currentTimeMillis();
```

Comment:
What's the difference?

Reply:
> What's the difference?

This was just a redundant conversion so I removed it